orderreturn/client.go

```diff
@@ -13,6 +13,18 @@ type Client struct {
 	Key string
 }
 
+// Get returns the details of an order return
+func Get(id string, params *stripe.OrderReturnResourceParams) (*stripe.OrderReturn, error) {
+  return getC().Get(id, params)
+}
+
+func (c Client) Get(id string, params *stripe.OrderReturnResourceParams) (*stripe.OrderReturn, error) {
+  path := stripe.FormatURLPath("/v1/order_returns/%s", id)
+  orderreturn := &stripe.OrderReturn{}
```

Comment:
Package names in Go are all lower case like `orderreturn`, but camel case for variable names please :)